e2e/testvalues/values.go

```diff
@@ -9,6 +9,9 @@ import (
 const (
 	StartingTokenAmount int64 = 10_000_000
 	IBCTransferAmount   int64 = 10_000
+	// ImmediatelyTimeout can be used as the value for timeout in nano seconds when we want
```

Comment:
nit: this might be redundant bc it will always be 1, and usually we should use these const values as something we might want to change update in the long run? maybe another way could be to set it as one and then add this comment you have describing the const in the code above where you use "1" as the TimeoutValue? not a blocker tho imo

Reply:
actually after thinking some more, maybe the clearest solution is to simply make the constant return

```golang
&ibc.IBCTimeout{
  NanoSeconds: 1,
}
```

This way the result can be passed directly into the `SendIBCTransfer` function. WDYT?

Reply:
The above would look like

```golang
chainATx, err = chainA.SendIBCTransfer(ctx, channelA.ChannelID, chainAWallet.KeyName, chainBWalletAmount, testvalues.ImmediatelyTimeout())
```

Reply:
yes! this lgtm :)